docs/example/saxml/leader-worker-set.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: leaderworkerset.x-k8s.io/v1
+kind: LeaderWorkerSet
+metadata:
+  name: saxml-multi-host
+  annotations:
+    leaderworkerset.sigs.k8s.io/exclusive-topology: cloud.google.com/gke-nodepool
+spec:
+  replicas: 2
+  leaderWorkerTemplate:
+    size: 9
+    restartPolicy: RecreateGroupOnPodRestart
+    leaderTemplate:
+      metadata:
+        labels:
+            app: saxml-admin
+      spec:
+        serviceAccountName: sax-sa
+        nodeSelector:
+          cloud.google.com/gke-tpu-topology: 4x8
+          cloud.google.com/gke-tpu-accelerator: tpu-v5-lite-podslice
+        tolerations: 
+        - key: "google.com/tpu"
+          operator: "Exists"
+          effect: "NoSchedule"
+        containers:
+        - name: sax-admin-server
+          image: us-docker.pkg.dev/cloud-tpu-images/inference/sax-admin-server:nightly
+          securityContext:
+            privileged: true
+          ports:
+          - containerPort: 10000
+          env:
+          - name: SAX_REPLICA_NAME
+            valueFrom:
+              fieldRef: 
+                fieldPath: metadata.name 
+          - name: SAX_GCS_BUCKET
+            value: GCS_BUCKET
+        - name: sax-http
+          image: gcr.io/tpu-vm-gke-testing/saxml-lws-testing-modified/httpserver-merged:latest
```

Comment:
will this be published in the same place as the other containers?

Reply:
yes it will